Sources/Components/VerificationView/VerificationView.swift

```diff
@@ -8,7 +8,7 @@ public protocol VerificationViewDelegate: AnyObject {
 
 public class VerificationView: UIView {
     private lazy var verificationImageView: UIImageView = {
-        let image = UIImage(named: .spidLogo)
+        let image = UIImage(named: .bankID)
         let imageView = UIImageView(image: image)
         imageView.clipsToBounds = true
         imageView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Should the imageView have `contentMode` set to `.scaleAspectFit`? I guess the image has the correct dimensions, but just in case? Then you could remove the line with `clipsToBounds` as well.

Reply:
Good catch🥇